context.go

```diff
@@ -323,13 +324,25 @@ func (c *context) getQueries() url.Values {
 // WrapF is an adapter for wrapping http.HandlerFunc and returns a HandlerFunc function.
 func WrapF(f http.HandlerFunc) HandlerFunc {
 	return func(c Context) {
+		if len(c.Params()) > 0 {
+			ctx := netcontext.WithValue(c.Ctx(), paramsKey, c.Params().Clone())
```

Comment:
The `ctx` variable is in conflict with the imported context line 8 `ctx "context"`

Reply:
Bien vu. Fixed.